python/lsst/ts/MTMount/client_server_pair.py

```diff
@@ -197,7 +197,7 @@ async def connect(self, port=None):
         if not self.client_port:
             raise ValueError(f"client_port={self.client_port!r} must be nonzero")
 
-        print(f"ClientServerPair waiting for client to connect")
+        self.log.debug(f"ClientServerPair waiting for client to connect")
```

Comment:
You can make the code a bit faster sparring f, which is not needed (no argument to include).

Reply:
Yes and flake8 now flags this, but not the version we have available in the DM stack or Docker develop containers.